### DISCUSSION TITLE
Render JSON data as a blog in Next.js

### DISCUSSION THREAD

**Discussion by bojjakaseenadhreddy** [codex-team/editor.js#2528] [Q&A]
I am planning to build a blog using Editor.js and Next.js

I was able to store the blocks data in the Database as JSON format.

My question here is, how to display the JSON data in the UI for the users to display as a blog?

I have tried `readOnly` flag, that seems not good. see below image.

<img width="1011" alt="Screenshot 2023-11-06 at 1 25 51 PM" src="https://github.com/codex-team/editor.js/assets/46647083/d7ecb2ba-b26d-4b8d-9d32-de554ee89e0d">

How I can render this JSON data to make it more user friendly.

Thanks

**Comment by sparklog** [comment]
try editorjs-html that I found in github(https://github.com/pavittarx/editorjs-html)

**Reply by bmiit145** [reply]
yea , It can be work fine !!

**Comment by bmiit145** [comment]
you can use a render package such as editorjs-html that can be best for render and view in any language
